pandas/tests/io/json/test_pandas.py

```diff
@@ -730,34 +718,33 @@ def test_reconstruction_index(self):
         result = read_json(df.to_json())
         tm.assert_frame_equal(result, df)
 
-    def test_path(self, float_frame):
+    def test_path(self, float_frame, int_frame, datetime_frame):
         with tm.ensure_clean("test.json") as path:
             for df in [
                 float_frame,
-                self.intframe,
-                self.tsframe,
-                self.mixed_frame,
```

Comment:
I just removed `mixed_frame` here as this is its only use in the module; doesn't seem important to re-create, and in its current state it was really the same thing as `float_frame` anyway